aspnetcore/security/identity-management-solutions.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Identity management solutions for .NET web apps
+description: A list of available products, packages, and services that enable identity management, including authentication and authorization, for ASP.NET Core web apps.
+author: JeremyLikness
+ms.author: jeliknes
+ms.date: 6/14/2023
+uid: security/identity-management-solutions
+---
+# Identity management solutions for .NET web apps
+
+The following table provides an overview of various identity management solutions that developers can use in their ASP.NET Core applications. These solutions offer features and capabilities to manage [user authentication](xref:security/authentication/index), [authorization](xref:security/authorization/introduction), and [user identity](xref:security/authentication/identity) within your application. It includes options for container-based solutions, self-hosted (you manage the installation and infrastructure to support it), and managed (cloud-based or "identity-as-a-service"). 
+
+ The table lists both open source and commercial solutions in alphabetical order, each accompanied by essential details such as license type, website, and documentation that is specific to ASP.NET Core integration. Explore the table below to discover the identity management solutions that best align with your application needs.
+
+> [!TIP]
+> Many of the commercial licenses provide "community" and/or free options that may be available depending on your company size and application requirements.
+
+|Name  |Type | License Type  |Website  |Docs  |
+|---------|-----|--------|---------|---------|
+|**ASP.NET Core Identity**| Self host |[OSS (MIT)](https://github.com/dotnet/aspnetcore/blob/main/LICENSE.txt)|[https://dotnet.microsoft.com/](https://dotnet.microsoft.com/apps/aspnet)|[Secure a web app with ASP.NET Core Identity](https://learn.microsoft.com/en-us/training/modules/secure-aspnet-core-identity/)|
```

Comment:
Do a global replace to `https://learn.microsoft.com/en-us` with ``
```suggestion
|**ASP.NET Core Identity**| Self host |[OSS (MIT)](https://github.com/dotnet/aspnetcore/blob/main/LICENSE.txt)|[https://dotnet.microsoft.com/](https://dotnet.microsoft.com/apps/aspnet)|[Secure a web app with ASP.NET Core Identity](/training/modules/secure-aspnet-core-identity/)|
```

Reply:
Yeah I have a few links to fix up, working on it now.